docs/articles/documentation/guides/basic-guides/assert.md

```diff
@@ -138,6 +133,45 @@ To set the timeout when you launch tests, pass the timeout value to the [runner.
 method if you use API or specify the [assertion-timeout](../../reference/command-line-interface.md#--assertion-timeout-ms) option
 if you run TestCafe from the command line.
 
+### Smart Assertion Limitations
+
+Smart assertions auto-retry only works with:
+
+* Promises returned from [ClientFunctions](../../reference/test-api/clientfunction/README.md),
+* [Selector](../../reference/test-api/selector/README.md) properties,
+* [RequestLogger.count](../../reference/test-api/requestlogger/count.md) properties,
+* [RequestLogger.contains](../../reference/test-api/requestlogger/contains.md) properties.
+
+The auto retry feature does not work with [DOM Node state](../../reference/test-api/domnodestate.md) properties. If you execute the selector as an asynchronous function, its value is immediately resolved, doesn't update and the corresponding test fails:
```

Comment:
Is "auto retry" a feature name (e.g., a button name in TestCafe)? If so, does it need to be capitalized (Auto-Retry feature")?

```suggestion
The auto-retry feature does not work with [DOM Node state](../../reference/test-api/domnodestate.md) properties. If you execute the selector as an asynchronous function, its value is immediately resolved, does not update, and the corresponding test fails:
```

Reply:
It's not really a name. just a reference